File: src/main/java/jp/developer/bbee/pcassem/presentation/data/ReviewItem.java
package jp.developer.bbee.pcassem.presentation.data;

import java.util.Locale;

public class ReviewItem {
    public String itemName;
    public int itemPrice;

    @Override
    public String toString() {
        return String.format(Locale.JAPAN, "%s(¥%,d)", itemName, itemPrice);
    }
}

File: src/main/java/jp/developer/bbee/pcassem/presentation/data/ReviewItem.kt
package jp.developer.bbee.pcassem.presentation.data

import java.util.Locale

class ReviewItem {
    @JvmField var itemName: String? = null
    @JvmField var itemPrice: Int = 0

    override fun toString() = String.format(Locale.JAPAN, "%s(¥%,d)", itemName, itemPrice)
}
